### DISCUSSION TITLE
gpg verify

### DISCUSSION THREAD

**Discussion by amicwall** [Ride-The-Lightning/c-lightning-REST#121] [General]
gpg --verify v0.7.2.tar.gz.asc v0.7.2.tar.gz
gpg: Signature made Mon 09 May 2022 03:37:59 PM PDT
gpg:                using RSA key 3E9BD4436C288039CA827A9200C9E2BC2E45666F
gpg: Can't check signature: No public key

where to get the public key ?

**Comment by saubyk** [comment]
Hi @amicwall you can get the PGP Key at: https://keybase.io/suheb

**Comment by saubyk** [comment]
Here's a command you can use to download and import the pub key:
`curl https://keybase.io/suheb/pgp_keys.asc | gpg --import`

**Reply by saubyk** [reply]
Moved the issue to discussions, in case you need more help. Thanks.
